src/inspector_io.cc

```diff
@@ -1,476 +1,445 @@
-#include "inspector_io.h"

```

Comment:
This is the result of `git checkout nodejs/master src/inspector_io.cc`.  At some point the line endings must have gotten messed up.  The diff here just removes the UTF-8/UTF-16 conversion workarounds that I added for inspector.